libs/payments/cart/src/gql/setup-cart.input.ts

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class SetupCartInput {
+  @Field()
```

Comment:
I'm not sure actually. Technically, they don't need it. For many of them it felt unnecessary, however for a few fields it could help clarify. So currently I'm opting for, where necessary.